Cognite.Simulator.Utils/ScopedRemoteApiSink.cs

```diff
@@ -18,11 +19,46 @@ namespace Cognite.Simulator.Utils {
     /// 
     public class ScopedRemoteApiSink : ILogEventSink
     {
-        private SimulatorLoggingConfig apiLoggerConfig;
+        private bool enabled;
+        private LogEventLevel minLevel;
+
         // Buffer for storing log data
         private readonly ConcurrentDictionary<long, List<SimulatorLogDataEntry>> logBuffer = new ConcurrentDictionary<long, List<SimulatorLogDataEntry>>();
         private long? defaultLogId;
+        
+        /// <summary>
+        /// Create a scoped api sink
+        /// </summary>
+        /// <param name="loggerConfig">The logger configuration</param>
+        public ScopedRemoteApiSink(LoggerConfig loggerConfig) : base(){
+            if (loggerConfig != null) {
+                enabled = loggerConfig.Remote == null || loggerConfig.Remote.Enabled;
+                if (enabled) {
+                    minLevel = ToSerilogLevel(loggerConfig.Remote?.Level);
+                }
+            }
+        }
 
+        private static LogEventLevel ToSerilogLevel(string level)
+        {
+            if (string.IsNullOrEmpty(level))
+            {
+                return LogEventLevel.Warning;
+            }
+            switch (level)
+            {
+                case "debug":
+                    return LogEventLevel.Debug;
+                case "error":
+                    return LogEventLevel.Error;
+                case "information":
+                    return LogEventLevel.Information;
+                case "warning":
+                    return LogEventLevel.Warning;
+                default:
+                    throw new ArgumentException($"Unknown log level: {level}");
```

Comment:
Check the parsing logic in extractors for setting the log level

Reply:
done